config/webpack.config.electron.lib.js

```diff
@@ -0,0 +1,18 @@
+var path = require('path');
```

Comment:
Can you explain why we need separate webpack  config for electron build? 

Ideally, if we can make it work without adding one more type of build, that will be simpler as the current build setup is already complicate enough.